crates/typst-library/src/compute/foundations.rs

```diff
@@ -82,6 +82,21 @@ pub fn panic(
     Err(msg)
 }
 
+/// Display: Warning
```

Comment:
And there should be some documentation above here too

Reply:
Addressed